var/spack/repos/builtin/packages/self/package.py

```diff
@@ -0,0 +1,50 @@
+# Copyright Spack Project Developers. See COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Self(CMakePackage):
```

Comment:
Can we call this `spectral-element-library-fortran` or something similar? `self` is not a keyword in Python, but better to avoid ambiguity.